polaris-react/src/components/Tooltip/Tooltip.tsx

```diff
@@ -86,10 +85,7 @@ export function Tooltip({
   onOpen,
   onClose,
 }: TooltipProps) {
-  const {polarisSummerEditions2023} = useFeatures();
-
-  const borderRadius =
-    borderRadiusProp || (polarisSummerEditions2023 ? '2' : '1');
+  const borderRadius = borderRadiusProp || '2';
```

Comment:
[This PR](https://5d559397bae39100201eedc1-tnpnnmnlft.chromatic.com/?path=/story/all-components-tooltip--border-radius)
[Production](https://storybook.polaris.shopify.com/?path=/story/all-components-tooltip--border-radius&globals=polarisSummerEditions2023:true)